frontend/app/controllers/concerns/spree/checkout/address_book.rb

```diff
@@ -42,11 +42,7 @@ def normalize_addresses
         if @order.bill_address_id != @order.ship_address_id && bill_address == ship_address
           @order.update_column(:bill_address_id, ship_address.id)
           bill_address.destroy
-        elsif params[:save_user_address]
```

Comment:
@KacperMekarski we still need to keep this option for older installations and for those who do not need to store that address, however, this should be by default true (let's add a hidden field tag with `true` value in the frontend checkout code)

Reply:
@damianlegawiec done.